### PR TITLE
[Swift 4] Add `createURLRequest` method

### PR DESCRIPTION
**Pull request by d-date**
### PR checklist

- [x] Read the [contribution guidelines](https://github.com/openapitools/openapi-generator/blob/master/CONTRIBUTING.md).
- [x] Ran the shell script under `./bin/` to update Petstore sample so that CIs can verify the change. (For instance, only need to run `./bin/{LANG}-petstore.sh` and `./bin/security/{LANG}-petstore.sh` if updating the {LANG} (e.g. php, ruby, python, etc) code generator or {LANG} client's mustache templates). Windows batch files can be found in `.\bin\windows\`.
- [x] Filed the PR against the [correct branch](https://github.com/OpenAPITools/openapi-generator/wiki/Git-Branches): `master`~~, `3.4.x`, `4.0.x`~~. Default: `master`.
- [x] Copied the [technical committee](https://github.com/openapitools/openapi-generator/#62---openapi-generator-technical-committee) to review the pull request if your PR is targeting a particular programming language.

### Description of the PR

Add utility method for return URLRequest

### CODE REVIEW

**Comment by d-date** [comment]
@jgavris (2017/07) @ehyche (2017/08) @Edubits (2017/09) @jaz-ah (2017/09) @d-date (2018/03)

**Comment by d-date** [comment]
@wing328 Could you check that? I think others are sleeping.

File: modules/openapi-generator/src/main/resources/swift4/AlamofireImplementations.mustache [hunk 1]
```diff
@@ -64,6 +64,12 @@ open class AlamofireRequestBuilder<T>: RequestBuilder<T> {
         return Alamofire.SessionManager(configuration: configuration)
     }
 
+    open func createURLRequest() -> URLRequest? {
```
Comment (wing328) [inline modules/openapi-generator/src/main/resources/swift4/AlamofireImplementations.mustache]: @d-date what about adding a 1-liner explaining what this function does?

Comment (d-date) [reply modules/openapi-generator/src/main/resources/swift4/AlamofireImplementations.mustache]: @wing328  Thanks! Added

**Pull request merged**